### PR TITLE
Add more wave colors

### LINKED ISSUE #76: Add more wave colors!
In [art.py](https://github.com/ryansurf/cli-surf/blob/main/src/art.py), there is a dictionary of colors a user can choose to color the wave ASCII art. Currently, there are only a handful of colors. It would be cool to add more options. This is the colors dictionary:

```
colors = {
    "end": "\033[0m",
    "default": "\033[39m",
    "red": "\033[0;31m",
    "green": "\033[0;32m",
    "yellow": "\033[0;33m",
    "blue": "\033[0;34m",
    "purple": "\033[0;35m",
    "teal": "\033[0;36m",
    "light_blue": "\033[0;34m",
}
```

More colors can be found [here](https://gist.github.com/JBlond/2fea43a3049b38287e5e9cefc87b2124). Add any color(s) you think may look good!

[help.txt](https://github.com/ryansurf/cli-surf/blob/main/help.txt) should also be updated to reflect the new color changes.

### PR DESCRIPTION
**Pull request by SeanAverS**
Fixes #76 

### Changes

`art.py`

- add new wave color `white` 
- add bold versions of existing wave colors 

`help.txt`

- update `Color Options` section to include newly added wave colors

### CODE REVIEW

**Comment by ryansurf** [comment]
Hey @SeanAverS, looks great! However, the linter is failing... can you run `make format` and `make lint` to make sure these are resolved? (if you don't have make installed, use the commands in the [makefile](https://github.com/ryansurf/cli-surf/blob/main/makefile))

You can ignore Pytest failing for Windows, thats an unrelated issue

**Comment by SeanAverS** [comment]
Sorry for the delay. Here are the test results. Let me know if you there's anything else that needs to be done and i'd be happy to help with it. 

76-cli-surf

File: .python-version
Comment (ryansurf) [inline .python-version:1]: We don't need this file

File: src/art.py
Comment (ryansurf) [inline src/art.py:1]: Nice!

**Comment by ryansurf** [comment]
Hey @SeanAverS,

Good work! I like the bold colors, I think I'll use them myself going forward.

One last thing - you commited the `.python-version` file, which we don't need. Can you remove that from the PR? Once that happens, I'll merge your changes.

Bonus points if you add `.python-version` to the `.gitignore`!

**Comment by ryansurf** [comment]
@all-contributors please add @SeanAverS for code

**Pull request merged**